src/test/scala/com/codacy/parsers/PhpUnitXmlParserTest.scala

```diff
@@ -0,0 +1,117 @@
+package com.codacy.parsers
+
+import java.io.File
+
+import com.codacy.parsers.implementation.PhpUnitXmlParser
+import org.scalatest.{BeforeAndAfterAll, EitherValues, Matchers, WordSpec}
+
+class PhpUnitXmlParserTest extends WordSpec with BeforeAndAfterAll with Matchers with EitherValues {
+  private val ROOT_PATH = "/home/codacy-php/"
+  private val VALID_REPORT = "src/test/resources/phpunitxml/index.xml"
+  private val INCORRECT_REPORT = "src/test/resources/phpunitxml/incorrect_index.xml"
+  private val COBERTURA_REPORT = "src/test/resources/test_cobertura.xml"
+  private val NON_EXISTENT_REPORT = "src/test/resources/non_existent_file.xml"
+  private val CONFIG_PHP_FILE = "Config.php"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name 'CONFIG_PHP_FILE' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/coverage-parser/pullRequest?prid=4907427)